src/platform/utilities/sso/index.js

```diff
@@ -34,3 +34,20 @@ export function checkAndUpdateSSOeSession() {
     }
   }
 }
+
+/*
+ * When set, the `requiresForceAuth` indicates that the current unauthenticated
+ * user will have to "force" authentication against SSOe in order to establish a
+ * session.
+ */
+export const getForceAuth = () => {
+  localStorage.getItem('requiresForceAuth');
+};
```

Comment:
I'm guessing `localStorage` is used to persist its value across page reloads?

Reply:
yeah; page reloads and new tabs.  Is it possible to use state to store this info?